File: Basic-Programs/Program21.c
#include<stdio.h>

int main()
{
    int iStandard = 0;
    
    printf("Enter Your Standard : \n");
    scanf("%d",&iStandard);
    
    switch(iStandard)
    {
        case 8:
            printf("Your Exam At 8am \n");
            break;
            
        case 9:
            printf("Your Exam At 9am \n");
            break;
            
        case 10:
            printf("Your Exam At 10am \n");    
            break;
            
        case 11:
            printf("Your Exam At 11am \n");
            break;
            
        default:
            {
                printf("Sorry we don't have information about your Standard \n");
            }        
    }
    
    return 0;
}

File: Basic-Programs/Program21.java
import java.util.Scanner;

class Program21
{
public static void main(String A[])
{
int iStandard = 0;

Scanner SC = new Scanner(System.in);  
    System.out.println("Enter Your Standard :");  
    iStandard = SC.nextInt();  
      
    switch(iStandard)  
    {  
        case 8:  
            System.out.println("Your Exam At 8am");  
            break;  
              
        case 9:  
            System.out.println("Your Exam At 9am");  
            break;     
              
         case 10:  
            System.out.println("Your Exam At 10am");  
            break;  
              
         case 11:  
            System.out.println("Your Exam At 11am");  
            break;     
              
         case 12:  
             System.out.println("Your Exam At 12pm");  
             break;   
               
          default:  
              System.out.println("Sorry! We don't have information about your Standard") ;  
              break;  
              
        }  
      
}

}

